### DISCUSSION TITLE
[Feature]: Show "x years ago" cards at the top of the timeline

### DISCUSSION THREAD

**Discussion by simon-vajda** [immich-app/immich#1603] [Feature Request]
### Feature detail

It would be nice to see older photos that were taken on the same day at the top of the timeline. Similar to the way Nextcloud Memories, or the _App-Which-Must-Not-Be-Named_ :) does it.

![image](https://user-images.githubusercontent.com/43452607/212221223-673445be-2faa-4366-aeea-40af6420319b.png)

### Platform

Web

**Comment by JVT038** [comment]
Wouldn't this be better for the new 'explore' tab instead of the main timeline?

**Reply by Kernald** [reply]
I feel like whatever the default tab (in this case, the timeline) is the best place to surface these - short-lived memories lose their interest significantly if they're not surfaced in a very obvious way.

**Comment by 19kmunz** [comment]
id like it if there were notifications for memories occasionally too!

some people like un-intrusive photo backup apps, while this feature is the only reason i havent switched to a new photo backup app

**Comment by C9Glax** [comment]
[This person](https://www.reddit.com/r/selfhosted/comments/145ixu3/alternative_for_google_photos_auto_creations/) is also looking for something similar.

**Comment by alextran1502** [comment]
Implemented in #2759
